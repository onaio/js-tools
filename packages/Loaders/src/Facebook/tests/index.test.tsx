import { mount } from 'enzyme';
import toJson from 'enzyme-to-json';
import 'jest-styled-components';
import React from 'react';
import { Facebook } from '..';

describe('Loaders.Facebook', () => {
  it('renders without crashing', () => {
    const wrapper = mount(<Facebook />);

    expect(toJson(wrapper.find('Facebook'))).toMatchSnapshot();
  });
});
