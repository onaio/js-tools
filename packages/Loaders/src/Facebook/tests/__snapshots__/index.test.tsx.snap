// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loaders.Facebook renders without crashing 1`] = `
.c0 {
  display: inline-block;
  position: relative;
  width: 80px;
  height: 80px;
}

.c0 div {
  display: inline-block;
  position: absolute;
  left: 8px;
  width: 16px;
  background: tomato;
  -webkit-animation: jirSBd 1.2s cubic-bezier(0,0.5,0.5,1) infinite;
  animation: jirSBd 1.2s cubic-bezier(0,0.5,0.5,1) infinite;
}

.c0 div:nth-child(1) {
  left: 8px;
  -webkit-animation-delay: -0.24s;
  animation-delay: -0.24s;
}

.c0 div:nth-child(2) {
  left: 32px;
  -webkit-animation-delay: -0.12s;
  animation-delay: -0.12s;
}

.c0 div:nth-child(3) {
  left: 56px;
  -webkit-animation-delay: 0;
  animation-delay: 0;
}

<Facebook
  animationSpeed={2.4}
  color="tomato"
  scaleSizeBy={1}
  scaleSpeedBy={1}
>
  <styled.div
    animationSpeed={2.4}
    color="tomato"
    scaleSizeBy={1}
    scaleSpeedBy={1}
  >
    <div
      className="c0"
      color="tomato"
    >
      <div />
      <div />
      <div />
    </div>
  </styled.div>
</Facebook>
`;
